### PR TITLE
Sites api addition

### PR DESCRIPTION
**Pull request by jorge-sepulveda**
added functionality to the stats endpoint in order to use the endpoint to create a more effective map for the covid data collection team.

### CODE REVIEW

File: libs/covid/data-api/src/lib/modules/lockdowns/lockdowns.service.ts [hunk 1]
```diff
@@ -343,3 +351,21 @@ export class LockdownsService extends BaseService<Lockdown> {
...
+}
+
+interface LockdownStat {
+  claims?: number;
+  lockdowns?: number;
+}
+
+export interface LockdownStats {
+  [key: string]: LockdownStats;
+}
```
Comment (Proantagonist) [inline libs/covid/data-api/src/lib/modules/lockdowns/lockdowns.service.ts:371]: These don't appear to be used anywhere. They have structure to the interfaces in the counties service.

Clean up any unused import statements.

File: libs/covid/data-api/src/lib/modules/counties/counties.service.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository, Like } from 'typeorm';
 
-import { County, CountyClaim, StatusType, EntityStatus, TestingSite } from '@tamu-gisc/covid/common/entities';
+import { County, CountyClaim, StatusType, EntityStatus, TestingSite, Lockdown } from '@tamu-gisc/covid/common/entities';
```
Comment (Proantagonist) [inline libs/covid/data-api/src/lib/modules/counties/counties.service.ts:5]: Cleanup unused imports

File: libs/covid/data-api/src/lib/modules/lockdowns/lockdowns.service.ts (hunk 1 above)
Comment (jorge-sepulveda) [reply libs/covid/data-api/src/lib/modules/lockdowns/lockdowns.service.ts]: deleted it

File: libs/covid/data-api/src/lib/modules/counties/counties.service.ts (hunk 2 above)
Comment (jorge-sepulveda) [reply libs/covid/data-api/src/lib/modules/counties/counties.service.ts:5]: Lockdown  is used in the ClaimWithData interface

Comment (Proantagonist) [reply libs/covid/data-api/src/lib/modules/counties/counties.service.ts:5]: `StatusType` is the odd one left out. That was my doing I think.

**Pull request merged**